common/views/components/PageLayout/PageLayout.tsx

```diff
@@ -138,11 +138,18 @@ const PageLayoutComponent: FunctionComponent<Props> = ({
   const socialPreviewCardImage = getCrop(image, '32:15') || image;
 
   const imageUrl =
-    (socialPreviewCardImage &&
-      convertImageUri(socialPreviewCardImage.contentUrl, 800)) ||
-    'https://i.wellcomecollection.org/assets/images/wellcome-collection-social.png';
```

Comment:
I added that image for the previous version of this, I think we could delete it now?

Reply:
done 👍